graphics/rendering/blender/README.md

```diff
@@ -0,0 +1,157 @@
+# Blender
+
+Blender is a free and open source software used for editing and rendering.
+
+<br>
+
+#### What you need:
+ - Singularity, which you can download and install from [here](https://github.com/sylabs/singularity).
+ - A text editor, like: `micro`, `vim` or `nano`.
+ - A blender image, like [these](https://free3d.com/3d-models/blender-scene)
```

Comment:
I'd use...   `https://www.blender.org/download/demo-files/`  ... Keep the URL local to the Blender site